e2e/ls_test.go

```diff
@@ -52,26 +56,29 @@ func TestListBucketsJSON(t *testing.T) {
 
 	cmd := s5cmd("--json", "ls")
 	result := icmd.RunCmd(cmd)
-
 	result.Assert(t, icmd.Success)
 
+	// find the first created bucket
+	startIdx := strings.Index(result.Stdout(), fmt.Sprintf(`"name":"%v-1"`, bucketPrefix))
```

Comment:
`result.Stdout()` is called multiple times. Please assign the value to a variable and reuse it.